pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -796,6 +799,31 @@ func (br *OVSBridge) GetPortList() ([]OVSPortData, Error) {
 	return portList, nil
 }
 
+// AllocateOFPort returns an OpenFlow port number which is not allocated or used by any existing OVS port. Note that,
+// the returned port is cached locally but not saved in OVSDB yet before the real port is created, so it might introduce
+// an issue for the conflict if it is occupied by others before OVS port creation.
```

Comment:
"an issue of conflict if the OFPort is occupied by another port creation."?

Reply:
Fixed. Thanks.